### DISCUSSION TITLE
first time npm run dev will caused missing page

### DISCUSSION THREAD

**Discussion by xiaods** [thuliteio/doks#1310] [General]
i follow the guide, and setup project with command:
npm create thulite@latest -- --template doks
it works like a charm,then wen i run first time cmd: npm run dev, it failed.
```
Error: add site dependencies: load resources: loading templates: "/Users/xiaods/Documents/code/getk8e.com/getk8e/node_modules/@thulite/doks-core/layouts/shortcodes/step.html:2:1": parse failed: template: shortcodes/step.html:2: function "page" not defined
```

**Comment by xiaods** [comment]
I finally found the issue: hugo version need update to upstream.

**Reply by h-enk** [reply]
Glad you got it sorted out. Yes, you need a recent version of Hugo — I will add this to the Doks docs (see also [Prerequisites](https://docs.thulite.io/install/auto/#prerequisites))
